content/blog/2022/2022-04-20-KaplanMeier.md

```diff
@@ -0,0 +1,73 @@
++++
+author = "Ela Praznik"
+date = "2022-04-15"
+draft = false
+title = "A gentle introduction to the Kaplan-Meier Estimator"
+type = "blog"
+thumbImage = ""
+frontPageImage = ""
+blog = [""]
+shortExcerpt = ""
+longExcerpt = ""
+x2images = true  # true if using retina screenshots, else false
++++
+
+The Kaplan-Meier Estimator is a function that relates time to the probability of an event of interest. It is widely used in survival analysis and with the addition of the Survival Analysis package to Orange it is now readily available for accessible visual analytics. However any kind of efficient data analysis requires a basic understanding of what the implemented model does. This blog post thus aims to:
+
+- explain the most important concepts needed for understanding the Kaplan-Meier Estimator,
+- show how to manually calculate and interpret a survival curve in order to built an intuitive understanding and
+- show how this can be implemented in Orange with a few clicks.
+
+**Why the need for a special method?**
+
+The "survival" in survival analysis gets it's name from it's use in the analysis of survival of patients. However the event of interest can be many things: a disease relapse, the first technical failure of a car or a user leaving a webpage. There are many real world problem where we would like to estimate the probability of an event happening in relation to the passing of time. For such estimations we need a survival dataset, which necessarily includes data instances characterized by the occurance of an event and the time at which it did. 
```

Comment:
gets it's --> gets its
from it's --> from its
There are many real world problem --> There are many real world problems
the occurance --> the occurrence